README.rst

```diff
@@ -1,65 +1,60 @@
-installation package from OpenVPN, and then manually install OpenVPN GUI.
+OpenVPN-GUI has been bundled with OpenVPN installers for a long time, so there
+is rarely a need to install it separately. Bleeding-edge
+versions of OpenVPN-GUI are available in `OpenVPN snapshot
+installers <http://build.openvpn.net/downloads/snapshots/>`_ based on Git master
+branch. OpenVPN-GUI gets installed by default in all OpenVPN installers.
 
+Installation using the official OpenVPN installers
+**************************************************
 
-Installation using the bundled OpenVPN package with OpenVPN GUI included
-************************************************************************
-
-* Download openvpn-2.0.X-gui-1.0.X-install.exe from 
-  http://openvpn.se
-
-* If you have a previous version of OpenVPN GUI installed, shut it down.
+* Download an `OpenVPN installer <https://openvpn.net/index.php/open-source.html>`_
+* If you have a previous version of OpenVPN GUI running, shut it down.
   Make sure it's closed by ALL logged on users.
 
-* Run the install program. During the installation you can choose if the GUI
-  should be started automatically at system startup. The default is yes.
-
-* Create a xxxx.ovpn config-file with your favorite texteditor and save it in:
-  C:\Program files\OpenVPN\config\. You should NOT use the "log" or "log-append"
-  options as OpenVPN GUI redirect the normal output to a log file itself. 
-  There is a sample config files in the "sample-config" folder. Please
-  refer to the OpenVPN project homepage for more information regarding 
-  creating the configuration file. http://openvpn.net/
-
+* Run the OpenVPN installer
 
 Manual installation of OpenVPN GUI
 **********************************
 
-* Download and install OpenVPN from http://openvpn.net/
+* `Download <https://openvpn.net/index.php/download/community-downloads.html>`_
+  and install OpenVPN
 
-* Download openvpn-gui-1.0.X.exe and save it in OpenVPN's bin folder.
-  Default is "C:\Program Files\OpenVPN\bin\". You must put it in this folder
+* Download OpenVPN GUI of your choice and save it in OpenVPN's bin folder.
+  Default is *C:\\Program Files\\OpenVPN\\bin\\*. You must put it in this folder
   because OpenVPN GUI depends on the OpenSSL DLLs installed in this folder by
   OpenVPN.
 
-* Create a xxxx.ovpn config-file with your favorite texteditor and save it in:
-  C:\Program files\OpenVPN\config\. You should NOT use the "log" or "log-append"
-  options as OpenVPN GUI redirect the normal output to a log file itself. 
-  There is a sample config files in the "sample-config" folder. Please
-  refer to the OpenVPN project homepage for more information regarding 
-  creating the configuration file. http://openvpn.net/
+Configuring OpenVPN GUI to start on Windows logon
+*************************************************
+
+Put a short-cut to openvpn-gui.exe in your *Start -> All Program -> StartUp*
```

Comment:
Yes, this is for the master branch. We probably need not touch README in the v10 branch, even though it's quite outdated even there.